bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIProvider.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
```

Comment:
Fair spot, but this is something which isn't really holding up this PR from going in could be addressed by automatic spotless fix once PR is merged (or while it is merged through robotized process). This is a generic point on multiple PRs which suffer from same issue as there is bunch of them for core and addons.
IANAL, but I need note that date in source file header is authoring date which is different for each file. Copyright can be established at time of making the sources, hence in theory it can't be "2010" for file which was made in 2021. It might be also doubtful if changing a comment which does not change contributed works. To make an example - re-printing of an book released in 1999 without changes in 2022 does not extend the copyright as the printing date is just an annotation on the covers, just like source file header.

Reply:
Sorry I normally don't merge PRs that will break builds.😉